delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/realms/RealmCommand.scala

```diff
@@ -0,0 +1,74 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.model.realms
+
+import akka.http.scaladsl.model.Uri
+import ch.epfl.bluebrain.nexus.delta.sdk.model.Identity.Subject
+import ch.epfl.bluebrain.nexus.delta.sdk.model.Label
+
+/**
+  * Enumeration of Realm command types.
+  */
+sealed trait RealmCommand extends Product with Serializable {
+
+  /**
+    * @return the id of the realm
+    */
+  def label: Label
+
+  /**
+    * @return the subject that intends to evaluate this command
+    */
+  def subject: Subject
+}
+
+object RealmCommand {
+
+  /**
+    * An intent to create a new realm.
+    *
+    * @param label        the label of the realm
+    * @param name         the name of the realm
+    * @param openIdConfig the address of the openid configuration
+    * @param logo         an optional address for a logo
+    * @param subject      the subject that intends to evaluate this command
+    */
+  final case class CreateRealm(
+      label: Label,
+      name: String,
```

Comment:
Shouldn't the name be `name: Name`? since the label is `label: Label`

Reply:
yep